etc/local_grapl/grapl_provision.py

```diff
@@ -291,6 +291,7 @@ def drop_all(client) -> None:
 
 
 if __name__ == '__main__':
+    time.sleep(10)
```

Comment:
I'd drop this a bit, since provisioning blocks other services from working. Maybe 2 or 3 seconds.

Connections to dgraph won't fail even if the server is in a bad state, and all of the operations should silently retry .

Reply:
I believe I set this because I was seeing errors connecting to either localstack (sqs) or minio. Like one of those two hadn't come up yet, but the provisioner was trying to run against them and failing. I'll see if I can turn this down a bit and have it still work.